pep-0695.rst

```diff
@@ -590,15 +599,10 @@ the defined type parameters.
 Accessing Type Parameters at Runtime
 ------------------------------------
 
-A new read-only attribute called ``__type_variables__`` is available on class,
-function, and type alias objects. This attribute is a tuple of the active
-type variables that are visible within the scope of that class, function,
-or type alias. This attribute is needed for runtime evaluation of stringified
-(forward referenced) type annotations that include references to type
-parameters. Functions like ``typing.get_type_hints`` can use this attribute
-to populate the ``locals`` dictionary with values for type parameters that
-are in scope when calling ``eval`` to evaluate the stringified expression.
-The tuple contains ``TypeVar`` instances.
+A new read-only attribute called ``__type_params__`` is available on generic class,
+function, and type alias objects. This attribute is a tuple of the
```

Comment:
It was initially unclear to me reading this that "generic class" was being used as a compound adjective(?) here. This is slightly more verbose, but might be clearer:

```suggestion
A new read-only attribute called ``__type_params__`` is available on generic class objects,
function objects, and type alias objects. This attribute is a tuple of the
```

Reply:
Maybe just "generic classes, functions, and type aliases"?

Reply:
Also works!